.github/workflows/ci.yml

```diff
@@ -108,6 +108,7 @@ jobs:
             libgraphicsmagick1-dev \
             libgtk-3-dev \
             libheif-dev \
+            libilmbase-dev \
```

Comment:
IlmBase != Imath

I don't think Imath >= 3.1 is available for Ubuntu 22.04? Mayba some PPA...

Reply:
I initially added an installation of libimath-dev, but due to the peculiarities of inter-package dependencies, this combination of packages could not be installed. So I replaced it with libilmbase-dev just to see what we get.

But, unfortunately, this will not help, because this way too old version of the library will be installed. I just wrote a comment about this and will probably close the PR, as this change adds a library that will not be used now and is not even suitable as pre-added for the next Ubuntu release (the package name will have to be changed).

Reply:
Probably because openexr is < 3